postgresqldb_log:
  dbname: "postgres"
  user: "postgres"
  pswd: "root"
  hostaddr: "127.0.0.1" 
  port: "5432"
  table_name: "table_birfen"
  table_columns: ["task_id","robot_id","signal_time","signal_type","signal_description"]
  table_update: {
      search_column: "signal_time",
      update_columns: "signal_description",
  }

